"Insert a Wazuh API entry automatically" page removed and its content moved - for branch 3.12 (#2563)
wazuh/wazuh-documentation: source/installation-guide/installing-splunk/splunk-app.rst

    .. image:: ../../images/splunk-app/app-icon.png
      :align: center

> 4. The app will redirect you to the *Settings* tab, where fill the form with your ``Wazuh API credentials``. Use the URL and port from your Wazuh API server.

Reviewer: ```suggestion
#. The app will redirect you to the *Settings* tab, where fill the form with your ``Wazuh API credentials``. Use the URL and port from your Wazuh API server.
```

Reviewer: We should try to use always `#.` for enumerations.

      :align: center
      :width: 100%

    If you want to add the ``Wazuh API credentials`` more quickly (for instance, for deployment purposes) you can execute the following command:

    .. code-block:: console

      # curl -X POST "http://<SPLUNK_IP>:<SPLUNK_PORT>/en-US/custom/SplunkAppForWazuh/manager/add_api?url=<WAZUH_API_URL>&portapi=<WAZUH_API_PORT>&userapi=<WAZUH_API_USERNAME>&passapi=<WAZUH_API_PASSWORD>"

    Note the following:

>     1. ``<SPLUNK_IP>`` is the **hostname or IP address** of the Splunk instance where the app was installed.

Reviewer: ```suggestion
    #. ``<SPLUNK_IP>`` is the **hostname or IP address** of the Splunk instance where the app was installed.
```

>     2. ``<SPLUNK_PORT>`` is the **port of the Splunk instance** where the app was installed. By default, it's 8000.

Reviewer: ```suggestion
    #. ``<SPLUNK_PORT>`` is the **port of the Splunk instance** where the app was installed. By default, it is 8000.
```

>     3. ``<WAZUH_API_URL>``, ``<WAZUH_API_PORT>``, ``<WAZUH_API_USERNAME>`` and ``<WAZUH_API_PASSWORD>`` represent the **Wazuh API credentials** to be stored on the app. Keep in mind that the Wazuh API URL must include ``http://`` or ``https://``, depending on the current configuration.

Reviewer: ```suggestion
    #. ``<WAZUH_API_URL>``, ``<WAZUH_API_PORT>``, ``<WAZUH_API_USERNAME>`` and ``<WAZUH_API_PASSWORD>`` represent the **Wazuh API credentials** to be stored on the app. Keep in mind that the Wazuh API URL must include ``http://`` or ``https://``, depending on the current configuration.
```
